### PR TITLE
feat: Adding the chat app

### PR DESCRIPTION
**Pull request by elviskahoro**
Adding an MVP of our revamped chat app

### CODE REVIEW

**Comment by elviskahoro** [comment]
* **#268** <a href="https://app.graphite.dev/github/pr/reflex-dev/reflex-examples/268?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @elviskahoro and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: chat_v2/chat_v2/page_chat/chat_state.py [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+            AI_MODEL = "meta-llama/Llama-3.2-90B-Vision-Instruct-Turbo"
+
+        case _:
+            raise ValueError("Invalid AI provider. Please set AI_PROVIDER environment variable")
+
+
+def get_otel_headers() -> None:
+    global OTEL_HEADERS
+    global OTEL_ENDPOINT
+    otel_provider = os.environ.get("OTEL_PROVIDER")
```
Comment (picklelo) [inline chat_v2/chat_v2/page_chat/chat_state.py:60]: We should make this optional / app should run without this set

**Comment by Alek99** [comment]
Do we want to keep the button in or comment them out because they are not functional?

**Comment by elviskahoro** [comment]
> Do we want to keep the button in or comment them out because they are not functional?

I'm guessing you mean the action bar. I can comment them out

**Pull request closed without merging**